common/types/bank.go

```diff
@@ -0,0 +1,349 @@
+package types
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+// Token definition
+type Token struct {
```

Comment:
Token is `tokens` not `bank` ;)

Reply:
I split the bank.go into account.go, coins.go and tokens.go then.  Please have a review again